### ISSUE TITLE
Bullets is showing on center of cell

### ISSUE THREAD

**Issue by bosiljkakostic** [amcharts/amcharts4#1189]
If I have date Axis, and interval year, bullet at '2019-01-01' and '2019-06-10', have no to be on the same place in the cell (in the center).  It is beyond one another. It has to be visible that it is not on the same dates. It works correctly in V3, but not in V4

**Comment by martynasma** [comment]
You'll need to set `baseInterval` to something other than a year if you want those to appear separately.

`baseInterval` is a way of instructing your chart about what granularity is of your data. Obviously it's not of yearly granularity, so you need to set it accordingly, e.g.:

```
dateAxis.baseInterval = {
  timeUnit: "month",
  count: 1
};
```

**Comment by bosiljkakostic** [comment]
I change it dynamically if some switch is on (month switch), but the picture stay same, even if I resize by scrollbars
![image](https://user-images.githubusercontent.com/48427836/55949517-140dcb00-5c53-11e9-8afa-df8bbbad2fab.png)

**Comment by bosiljkakostic** [comment]
![image](https://user-images.githubusercontent.com/48427836/55949595-3dc6f200-5c53-11e9-9229-b74475f439d7.png)

**Comment by bosiljkakostic** [comment]
Regardless of the base interval, it has no to be on the center of the cell, I have no place on date Axis to put days as the base interval (or even months), but I do not want those bullets to be one beyond other, it has to be placed relatively within the cell with date dependency

**Comment by bosiljkakostic** [comment]
Try to use some long period, and set a small interval, it will not change anything, even when you resize some part of the chart, bullets will be one over another. In V3 it works correctly, here it can not be done correctly, nevertheless used interval

**Comment by martynasma** [comment]
Can you post a codepen of your chart so that I can tinker with it?

**Comment by bosiljkakostic** [comment]
Thanks, I found a solution when I start made example in JSFiddle.
The problem was that I was put the only chart.dateFormatter.inputDateFormat = "yyyy-MM-dd"; when I set the same for series, it started to work well (with series.dateFormatter.inputDateFormat = "yyyy-MM-dd"; it is O.K., now. My data is connected only on the chart (not separate for series). Before that, all dates were in January, and hours was months.
